pkgs/by-name/gn/gnome-maps/package.nix

```diff
@@ -68,6 +69,13 @@ stdenv.mkDerivation (finalAttrs: {
     libsoup_3
   ];
 
+  mesonFlags = [
+    "--cross-file=${writeText "crossfile.ini" ''
+      [binaries]
+      gjs = '${lib.getExe gjs}'
```

Comment:
> The eval error is because of this depends on [240d44b](https://github.com/NixOS/nixpkgs/commit/240d44bc7475f47a88d0b4cdcb1da341dcf2716d).

Thanks. Cherry-picked now.